types/html-to-rtf/html-to-rtf-tests.ts

```diff
@@ -0,0 +1,29 @@
+import * as htmlToRtf from 'html-to-rtf';
+
+const html = `
+<h1>Title <span style="color:rgb(255,0,0);">with</span> tag h1<h1>
+<div>
+	<p style="color:#333; margin:5px;" class="test" align="center">
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/html-to-rtf/html-to-rtf-tests.ts;
  Line: 6;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.